tests/api-diff/run.t

```diff
@@ -28,3 +28,44 @@ there should be no diff:
 
   $ api-diff ref.cmi ref.cmi
   API unchanged!
+
+# Adding a module
+  $ cat > mod_add.mli << EOF
+  > module Hello :
+  >   sig
+  >     type t = unit = ()
+  >     val equal : t -> t -> bool
+  >     val compare : t -> t -> int
+  >     val to_string : t -> string
+  >   end
+  > EOF
+
+  $ ocamlc mod_add.mli
+
+  $ api-diff ref.cmi mod_add.cmi
+  Api changed!
+
+#Removing a module
+  $ cat > mod_remove.mli << EOF
```

Comment:
Here you're just copying the original `ref.mli` file so you're not removing anything.

It's probably best if you define `mod_ref.mli` and `mod_ref.cmi` files that define a couple of modules so you can use them as a base for the three test cases here.